client/src/modules/holidays/holidays.routes.js

```diff
@@ -0,0 +1,42 @@
+angular.module('bhima.routes')
+.config(['$stateProvider', function ($stateProvider) {
+  $stateProvider
+    .state('holidays', {
+      url         : '/holidays',
+      controller  : 'HolidayManagementController as HolidayCtrl',
+      templateUrl : 'modules/holidays/holidays.html',
+    })
+
+    .state('holidays.create', {
+      url : '/create',
+      params : {
+        holiday : { value : null },
```

Comment:
Do you ever use `$state.params.holiday`?   I couldn't find it.  If not, go ahead any remove these from both the edit and create state.